src/zope/sendmail/tests/test_delivery.py

```diff
@@ -46,6 +46,8 @@ def testInterface(self):
         verifyObject(IDataManager, manager)
         self.assertEqual(manager.callable, object)
         self.assertEqual(manager.args, (1, 2))
+        # required by IDataManager
+        self.assertTrue(isinstance(manager.sortKey(), str))
```

Comment:
You could use [assertIsInstance()](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance) here.